crates/ruff/tests/snapshots/show_settings__display_default_settings.snap

```diff
@@ -189,7 +189,7 @@ linter.rules.should_fix = [
 linter.per_file_ignores = {}
 linter.safety_table.forced_safe = []
 linter.safety_table.forced_unsafe = []
-linter.target_version = Py37
+linter.target_version = PythonVersion { major: 3, minor: 7 }
```

Comment:
I think this should serialize to "3.7" instead to have it symmetric with the deserialization.